README.md

```diff
@@ -21,9 +21,15 @@ Thermite is a Rake-based helper for building and distributing Rust-based Ruby ex
 2. In your gemspec, add `Rakefile` to the specification's `extensions` list.
 3. In `Rakefile`, add `require 'thermite/tasks'` and then add the tasks to the file by running:
 
-```ruby
-Thermite::Tasks.new
-```
+  ```ruby
+  Thermite::Tasks.new
+  ```
```

Comment:
Yes, I added whitespace before the code block to preserve numeration of `3.` and `4.`